crates/rspack_binding_options/src/options/raw_optimization.rs

```diff
@@ -48,8 +52,8 @@ impl RawOptionsApply for RawOptimizationOptions {
       plugins.push(split_chunks_plugin);
     }
     let chunk_ids_plugin = match self.chunk_ids.as_ref() {
-      "named" => NamedChunkIdsPlugin::new(None, None).boxed(),
-      "deterministic" => DeterministicChunkIdsPlugin::default().boxed(),
+      "named" => NamedChunkIdsPlugin::new(delimiter, None).boxed(),
+      "deterministic" => DeterministicChunkIdsPlugin::new(delimiter, None).boxed(),
```

Comment:
split_chunks.automatic_name_delimiter shouldn't affect how chunk id generated, webpack doesn't, it should affect how split_chunks_plugin works

Reply:
Yes, but It seems that rspack names the output files depending on chunk id partly. I have posted an example above: https://github.com/web-infra-dev/rspack/pull/4365#issuecomment-1770433366. Maybe I have a wrong understanding of how `automatic_name_delimiter` affects the generated files' name. Could you give me some tutorials? Thanks!

Reply:
> split_chunks.automatic_name_delimiter shouldn't affect how chunk id generated, webpack doesn't, it should affect how split_chunks_plugin works

I think I get it. But in rspack, it involves some troubled arguments passing, I will try it later. =w=